src/resources/snippets.ts

```diff
@@ -164,8 +164,8 @@ export default [
   {
     "title": "Rotating Cover Art",
     "description": "Adds circular mask to cover art and rotation",
-    "code": `@keyframes rotating { from { transform: rotate(0deg); } to { transform: rotate(360deg); }} .cover-art { animation: rotating 2s linear infinite; clip-path: circle(50% at 50% 50%);}`,
-    "preview": "resources/assets/snippets/rotating-coverart.png",
+    "code": `@keyframes rotating { from { transform: rotate(0deg); } to { transform: rotate(360deg); }} .FqmFsMhuF4D0s35Z62Js, .zmOtW0vqqn1qpZrtQ_w9 { animation: rotating 10s linear infinite; clip-path: circle(50% at 50% 50%);} .H0HbpIM3UrcupWIAjLWu { background: none; } .H0HbpIM3UrcupWIAjLWu { background: none; } .main-coverSlotCollapsed-container { padding-right: 10px !important; }`,
```

Comment:
These classes are already mapped in latest `css-map` commits and the snippet still works as expected.

https://github.com/spicetify/spicetify-marketplace/assets/77577746/57e8c427-fe89-4ec4-9615-93ad2b19454d

Run `spicetify restore backup apply` to fetch the new mappings

Reply:
oh ok thank you